v3/tasks/20250313-v3-quantamm/readme.md

```diff
@@ -0,0 +1,19 @@
+# 2025-03-13 - QuantAMM
+
+Deployment of the `QuantAMM` infrastructure for Balancer V3.
+
+The update weight runner orchestrates calling registred rules for pools. These rules determine weight vectors for target pools using chainlink oracles. The update weight runner then sets these weights on the pools. 
+
+The QuantAMMWeightedPoolFactory is a factory for those pools and uses the V2 split factory method. 
+
+## Useful Files
+
+- [Code](https://github.com/QuantAMMProtocol/QuantAMM-V1).
```

Comment:
Note that this will point to the exact commit when you have it. (Could use a placeholder at current main, and then update it.)

```suggestion
- [Code](https://github.com/QuantAMMProtocol/QuantAMM-V1/commit/4b0f162fcb1a63846d0153e9610504244858aba0).
```